meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -365,6 +366,18 @@ RESIN_CONFIGS[redsocks] = " \
     CONFIG_NETFILTER_XT_TARGET_REDIRECT=m \
     "
 
+# disable some large and commonly enabled configs to reduce image size
+RESIN_CONFIGS[reduce-size] = " \
+    CONFIG_OCFS2_FS=n \
+    CONFIG_GFS2_FS=n \
+    CONFIG_REISERFS_FS=n \
+    CONFIG_NTFS_FS=n \
```

Comment:
Hm. This is the one I'm worried that users might take advantage of. @shaunmulligan @petrosagg what do you think?

Reply:
NTFS? The in-kernel driver is (effectively) read-only and very lightly maintained. AFAIK everyone doing NTFS work on Linux is using NTFS-3G with FUSE.

Reply:
Good point. I thought they added support for writing to NTFS.